TESTS/mbed_drivers/echo/main.cpp

```diff
@@ -48,12 +53,10 @@ utest::v1::status_t greentea_failure_handler(const Case *const source, const fai
 
 Case cases[] = {
     Case("Echo server: x16", test_case_echo_server_x<16>, greentea_failure_handler),
-    Case("Echo server: x32", test_case_echo_server_x<32>, greentea_failure_handler),
-    Case("Echo server: x64", test_case_echo_server_x<64>, greentea_failure_handler),
```

Comment:
Seems ok, I'm guessing if any failures were to crop up in these cases it'd be a DAPlink related issue.

cc @c1728p9